src/TopDownProteomics/ProForma/ProFormaAmbiguityDescriptor.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TopDownProteomics.ProForma
+{
+    /// <summary>
+    /// Member of a tag annotating ambiguous localization of a modification along a proteoform sequence.
+    /// </summary>
+    public class ProFormaAmbiguityDescriptor
+        : ProFormaDescriptor
```

Comment:
I think inheritance is a fine way to go for starters. Once we push ambiguity up to the ProteoformGroup level and start reasoning about ranges, we will quickly see if this was the right decision.

Reply:
Agreed. :)